Source/WebCore/loader/PingLoader.cpp

```diff
@@ -184,6 +184,7 @@ void PingLoader::sendViolationReport(LocalFrame& frame, const URL& reportURL, Re
         request.setHTTPContentType("application/csp-report"_s);
         break;
     case ViolationReportType::COEPInheritenceViolation:
+    case ViolationReportType::CSPHashReport:
     case ViolationReportType::CORPViolation:
```

Comment:
Case sensitive alphabetical sorting would put CSP after CORP. Also noticed that _inheritance_ is misspelled here, with an _e_ instead of an _a_.